source/adapters/level_zero/memory.cpp

```diff
@@ -2020,7 +2022,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urMemImageGetInfo(
   std::ignore = PropSize;
   std::ignore = ImgInfo;
   std::ignore = PropSizeRet;
-  urPrint("[UR][L0] %s function not implemented!\n", __FUNCTION__);
+  logger::debug("{} function not implemented!", __FUNCTION__);
```

Comment:
Some of this kind of messages (`function not implemented`) are of the `debug` type whereas most are of the `error` type (in other files, too). Shouldn't they be unified and have the same log level across the level zero?

Reply:
Done